__mocks__/isomorphic-dompurify.js

```diff
@@ -0,0 +1,5 @@
+const mock = {
```

Comment:
The mock doesn't need to be done here. You can do it directly on the test. Since it's the only place we need to mock it for now :)

(I actually have the mock actually be done there, as it allows me to remember why this mock exists)